lib/elixir/pages/getting-started/basic-types.md

```diff
@@ -167,6 +167,8 @@ iex> !nil
 true
 ```
 
+Since only `false` and `nil` are considered "falsy", values like `0` and `""` that some other programming languages consider to also be "falsy" are in fact "truthy" in Elixir.
```

Comment:
We already mention false and nil right before, so what about this:

```suggestion
Similarly, values like `0` and `""`, which some other programming languages consider to also be "falsy", are also "truthy" in Elixir.
```

Reply:
Sounds good.